docs/pressrelease-wot-rec.html

```diff
@@ -52,8 +52,7 @@ <h2>Web technologies enable the full potential of the Internet of Things by inte
     integration across Internet of Things platforms and
     applications. 
     </p>
-    <p>Interoperability of devices and services for the Internet of Things (IoT) with cloud platforms and services 
-    at a worldwide scale is a huge challenge.<br> 
+    <p><br> 
```

Comment:
Please don't just delete text but rather improve it to address the additional interoperability challenges you are facing.

Reply:
done.

Reply:
there is an alternative text now. please review.